platform/ios/src/MGLMapView.mm

```diff
@@ -2698,17 +2755,29 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             id <MGLAnnotation> annotation = [self annotationWithTag:annotationTag];
             NSAssert(annotation, @"Unknown annotation found nearby tap");
             
-            MGLAnnotationImage *annotationImage = [self imageOfAnnotationWithTag:annotationTag];
-            if ( ! annotationImage.enabled)
+            MGLAnnotationContext annotationContext = _annotationContextsByAnnotationTag[annotationTag];
+           
+            CGRect annotationRect;
+            if (annotationContext.annotationView)
+            {
+                annotationRect = annotationContext.annotationView.frame;
```

Comment:
Yes. I have a task in this PR to investigate and refactor to use view hit testing. Will fix.